avcodec.go

```diff
@@ -58,6 +63,51 @@ func (d *avCodecDecoder) Description() string {
 	return fmt
 }
 
+func (d *avCodecDecoder) IsStreamable() bool {
```

Comment:
neat!

let's move this into avcodec.cpp, maybe called from avcodec_decoder_create? we should also stash this as a flag somewhere so we don't potentially recompute it

Reply:
sounds good, this approach came together nicely!